packages/back-end/src/models/FeatureRevisionModel.ts

```diff
@@ -302,12 +334,98 @@ export async function markRevisionAsPublished(
   );
 }
 
+export async function markRevisionAsReviewRequested(
+  revision: FeatureRevisionInterface,
+  user: EventAuditUser,
+  comment?: string
+) {
+  const action = "Review Requested";
+
+  const log: RevisionLog = {
+    action,
+    subject: "",
+    timestamp: new Date(),
+    user,
+    value: JSON.stringify(comment ? { comment } : {}),
+  };
+  migrateToPendingReview(revision.organization, revision.featureId);
+  await FeatureRevisionModel.updateOne(
+    {
+      organization: revision.organization,
+      featureId: revision.featureId,
+      version: revision.version,
+    },
+    {
+      $set: {
+        status: "pending-review",
+        publishedBy: user,
+        datePublished: new Date(),
+        dateUpdated: new Date(),
+        comment: comment ?? revision.comment,
```

Comment:
For both of these new functions, I'm not sure if we should be updating the revision comment.  If someone reviews and says `lgtm` as the comment, we wouldn't want to override the top-level revision's comment that describes the changes in details.  The review comment already exists in the log